### ISSUE TITLE
initLetsencrypt.sh downloads recommended settings twice

### ISSUE THREAD

**Issue by twatzl** [wmnnd/nginx-certbot#2]
Hello,

the initLetsencrypt.sh downloads the recommended settings twice from github. First in Line 25 and then again in Line 37. 

Wouldn't it make sense to download them only once to a temporary directory and then copy them over?
Or at least print the same info message for both?
Or is there any difference that I have overlooked?

Kind Regards

**Comment by unimariJo** [comment]
Just the bug, I have fixed it in my PR #1 😅

**Comment by wmnnd** [comment]
Hey @twatzl, thank you for reporting this!
@unimariJo has indeed fixed this problem in their PR which I have now merged into master. No more double-downloading :grinning:
